From bf5b40adf70d3f2ddafe1868682e35b52a63c31a Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 17 Apr 2018 09:12:40 +0800
Subject: [PATCH] arm64: dts: rockchip: Enable dmc for rk3308-evb-10 board

The ATF must contain the following commit:
35e0de42e8ac ("plat: rk3308: dfs: add ddr_get_rate implement")

Change-Id: Idf22418b830c9a1f4ebd2c9b19dc7345bc4cf1c6
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 48f6c75346e5..5e1efe012146 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -256,6 +256,11 @@
 	cpu-supply = <&vdd_core>;
 };
 
+&dmc {
+	center-supply = <&vdd_core>;
+	status = "okay";
+};
+
 &emmc {
 	bus-width = <8>;
 	cap-mmc-highspeed;
-- 
2.35.3

